samplesheets/views_api.py

```diff
@@ -69,7 +73,10 @@
 MD5_RE = re.compile(r'([a-fA-F\d]{32})')
 APP_NAME = 'samplesheets'
 IRODS_QUERY_ERROR_MSG = 'Exception querying iRODS objects'
-IRODS_EX_MSG = 'iRODS data request failed'
+IRODS_REQUEST_EX_MSG = 'iRODS data request failed'
+IRODS_TICKET_EX_MSG = 'iRODS access ticket failed'
+IRODS_TICKET_NO_UPDATE_FIELDS_MSG = 'No fields to update'
+IRODS_TICKETS_NOT_FOUND_MSG = 'No iRODS access tickets found'
```

Comment:
I believe the last constant here isn't used anymore, you might also want to review the usage of the rest.